### ISSUE TITLE
Bring back auto camel case but explicit this time

### ISSUE THREAD

**Issue by ahayzen-kdab** [KDAB/cxx-qt#1105]
Instead of implicit auto camel conversion make it explicit and potentially per extern block?

This could look like at a bridge level or extern block level.

```rust
#[cxx_qt::bridge(auto_camel_case)]
mod ffi {
  unsafe extern "C++Qt" {
    fn my_function(&self: MyQObject);  // would be myFunction in C++
  }
}
```

Or

```rust
#[cxx_qt::bridge]
mod ffi {
  #[auto_camel_case]
  unsafe extern "C++Qt" {
    fn my_function(&self: MyQObject);  // would be myFunction in C++
  }
}
```

**Comment by ahayzen-kdab** [comment]
The name could also include `cxx` in it such as `cxx_auto_camel`, as the way this works is if there is no cxx_name specified it will camel case the effective rust name.

**Comment by BenFordTytherington** [comment]
Implemented and merged in #1108
